app/components/version_milestones_component.html.erb

```diff
@@ -1,11 +1,5 @@
-<tr class="d-none last_step<%= version %>">
- <td><span style="cursor: pointer;" onclick="document.querySelectorAll('.version<%= version %>').forEach((elem) => elem.classList.toggle('d-none'));document.querySelector('.last_step<%= version %>').classList.toggle('d-none')">+</span><%= version %></td>
- <td><%= steps['accessioned'][:display] || steps.keys.first.titleize %></td>
- <td><%= steps['accessioned'][:time].nil? ? 'pending' : steps['accessioned'][:time].in_time_zone.to_s %></td>
-</tr>
```

Comment:
This is the row that is only displayed when the event is collapsed. It can no longer be collapsed.